src/components/VideoPlayer.vue

```diff
@@ -170,6 +174,27 @@ export default {
                             videoEl.currentTime = videoEl.currentTime + 5;
                             e.preventDefault();
                             break;
+                        case "ctrl+left": {
+                            videoEl.currentTime = chapters.findLast(
+                                chapter => chapter.start < videoEl.currentTime,
+                            ).start;
+                            e.preventDefault();
+                            break;
+                        }
+                        case "ctrl+right": {
+                            videoEl.currentTime =
+                                chapters.find(chapter => chapter.start > videoEl.currentTime).start || videoEl.duration;
+                            e.preventDefault();
+                            break;
```

Comment:
```suggestion
                            videoEl.currentTime = self.video.chapters.findLast(
                                chapter => chapter.start < videoEl.currentTime,
                            ).start;
                            e.preventDefault();
                            break;
                        }
                        case "ctrl+right": {
                            videoEl.currentTime =
                                self.video.chapters.find(chapter => chapter.start > videoEl.currentTime).start || videoEl.duration;
                            e.preventDefault();
                            break;
```
That variable isn't needed at all.